externalAPI/tradeshiftAPI/supplierAPI.go

```diff
@@ -95,27 +94,30 @@ func (t *TradeshiftAPI) SearchOffer(name string) (map[string]interface{}, error)
 	return r, err
 }
 
+func buildAdvancedSearchValue(name string) string {
+	return fmt.Sprintf("{\"name\":\"%v\"}", url.QueryEscape(name))
+}
+
 func (t *TradeshiftAPI) CreateOffer(name string, buyerID string) (string, error) {
 	method := "/product-engine/supplier/supplier/v1/offers"
 	params := []rest.UrlParam{
 		{
 			Key:   "buyerId",
-			Value: buyerID,
+			Value: url.QueryEscape(buyerID),
```

Comment:
Are you sure about encoding here? It looks like value is encoded in url query builder.

Reply:
this param is query part..

Reply:
discussed and fixed

Reply:
How it was fixed?

Reply:
fixed again